chapter15/kds/ex1.pl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+use Scalar::Util qw(looks_like_number);
+use 5.010001;
+
+my $random = int(1 + rand 100);
+
+while(1){
+  print "Please guess the number: ";
+  chomp( my $input = <STDIN> );
+  die "quitting..." if( $input eq  "exit" || $input eq "quit" || $input eq "" );
+  next if( !looks_like_number( $input ) );
```

Comment:
You could also add the `looks_like_number` check and the quitting dialogue to the given-when